src/pages/HistoryPage/CurrentEpochPanel.tsx

```diff
@@ -38,6 +38,7 @@ type Props = {
   isAdmin: boolean;
   css?: CSS;
   children?: React.ReactNode;
+  targetEpoch: boolean;
```

Comment:
what does this prop mean? like if it is targetted it should be open? @sshmm

Reply:
yes, Do you think another name will be more fit here?

Reply:
yes, it seems you are trying to have a boolean of whether this panel should be expanded or not. targetEpoch is context that doesn't exist within this component, or have meaning as a boolean to me.

How aobut we name it `expanded: boolean` which indicates whether the component should render as expanded or not. @sshmm